### PR TITLE
Update vn.m3u

### LINKED ISSUE #13130: Add: Bac Kan TV [VN]
### Channel ID

BacKanTV.vn

### Stream URL (optional)

http://123.31.36.68/live.m3u8?c=vstv360&q=high&type=tv&token=LX-ibJYRUq9pflRtYAxfYQ&time=1586395820&gcId=1532&userId=&deviceId=&deviceType=&location=NA&requestTime=1586309420781&pkg=pkg11.hni

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13131: Add: Bac Lieu TV [VN]
### Channel ID

BacLieuTV.vn

### Stream URL (optional)

https://tv.thbl.vn/live/tv/tv.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13132: Add: Cao Bang TV [VN]
### Channel ID

CaoBangTV.vn

### Stream URL (optional)

http://118.107.85.4:1935/live/smil:CRTV.smil/playlist.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13133: Add: Dien Bien TV [VN]
### Channel ID

DienBienTV.vn

### Stream URL (optional)

https://truyenhinh.vnptvas.vn/live.m3u8?c=vstv362&q=high&type=tv&token=LX-ibJYRUq9pflRtYAxfYQ&time=1586395820&gcId=1532&userId=&deviceId=&deviceType=&location=NA&requestTime=1586309420781&pkg=pkg11.hni

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13134: Add: Ha Giang TV [VN]
### Channel ID

HaGiangTV.vn

### Stream URL (optional)

http://123.31.36.68/live.m3u8?c=vstv365&q=high&type=tv&token=LX-ibJYRUq9pflRtYAxfYQ&time=1586395820&gcId=1532&userId=&deviceId=&deviceType=&location=NA&requestTime=1586309420781&pkg=pkg11.hni

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13135: Add: Hau Giang TV [VN]
### Channel ID

HauGiangTV.vn

### Stream URL (optional)

https://60acee235f4d5.streamlock.net/HGTV/d1/playlist.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13136: Add: Khanh Hoa TV [VN]
### Channel ID

KhanhHoaTV.vn

### Stream URL (optional)

http://210.245.20.94/hls/ktv1.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13137: Add: Kon Tum TV [VN]
### Channel ID

KonTumTV.vn

### Stream URL (optional)

https://tv.kontumtv.vn/live/kontumtv/kontumtv.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13138: Add: Lao Cai TV [VN]
### Channel ID

LaoCaiTV.vn

### Stream URL (optional)

https://cdn.3ssoft.vn/livetv/laocaitv/laocaitv/index.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by freearhey**
Added new links from issues.

Closes #13138, closes #13137, closes #13136, closes #13135, closes #13134, closes #13133, closes #13132, closes #13131, closes #13130

### CODE REVIEW

**Review by Carlinhos027 — DISMISSED** [review]

File: streams/vn.m3u [hunk 1]
```diff
@@ -147,3 +147,21 @@ https://live.baoquangninh.vn/qtvlive/tv3live.m3u8
 https://live.sonlatv.vn/sltvlive/tv1live-transmux.m3u8
 #EXTINF:-1 tvg-id="TuyenQuangTV.vn",Tuyen Quang TV (540p)
 https://live.tuyenquangtv.vn/hls/ttv.m3u8
+#EXTINF:-1 tvg-id="LaoCaiTV.vn",Lao Cai TV (1080p)
+https://cdn.3ssoft.vn/livetv/laocaitv/laocaitv/index.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/vn.m3u:151]: Either Geo or Not 24/7

Comment (freearhey) [reply streams/vn.m3u:151]: Updated

**Review by PopeyeTheSai10r — APPROVED** [review]

**Review by Sphinxroot — APPROVED** [review]

**Review by Carlinhos027 — APPROVED** [review]

File: streams/vn.m3u [hunk 2]
```diff
@@ -163,3 +163,5 @@ https://truyenhinh.vnptvas.vn/live.m3u8?c=vstv362&q=high&type=tv&token=LX-ibJYRU
 http://118.107.85.4:1935/live/smil:CRTV.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="BacLieuTV.vn",Bac Lieu TV (1080p)
 https://tv.thbl.vn/live/tv/tv.m3u8
+#EXTINF:-1 tvg-id="BacKanTV.vn",Bac Kan TV (406p)
+http://123.31.36.68/live.m3u8?c=vstv360&q=high&type=tv&token=LX-ibJYRUq9pflRtYAxfYQ&time=1586395820&gcId=1532&userId=&deviceId=&deviceType=&location=NA&requestTime=1586309420781&pkg=pkg11.hni
```
[the remark this answers is not in the document]
Comment (freearhey) [reply streams/vn.m3u:167]: maybe

[the remark this answers is not in the document]
Comment (R4IN2007) [reply streams/vn.m3u:167]: Stream works fine without the token.
Screenshot_20240601_051602_Chrome

**Pull request merged**